### ISSUE TITLE
New directory browsing outputting absolute file path on Windows

### ISSUE THREAD

**Issue by boughtonp** [eclipse/jetty.project#3549]
This is likely related to #3319 - on Windows, v9.4.16 has full paths instead of just the name.

The same release on Debian works fine, as does v9.4.15 on Windows (with older markup).

Example:
```
mkdir -p webapps/browse
touch webapps/browse/test.txt
java -jar ../jetty-distribution-9.4.16.v20190411/start.jar --module=deploy --module=http
```

Browsing to `http://localhost:8080/browse/` on Windows gives a dir list including:

`<a href="/browse/S:%5Cjetty%5Cdirectory-browsing%5Cwebapps%5Cbrowse%5Ctest.txt">S:\jetty\directory-browsing\webapps\browse\test.txt&nbsp;</a>`

On Debian it correctly outputs:

`<a href="/browse/test.txt">test.txt&nbsp;</a>`

**Comment by joakime** [comment]
The filename (on the Resource path) isn't being detected properly on windows. ugh.

**Comment by joakime** [comment]
Opened PR #3552

**Comment by joakime** [comment]
Work completed in `jetty-9.2.x` and merged up to `jetty-9.4.x`.
This fix will be present in the next releases of each.
